Specialization/Blood.lua

```diff
@@ -124,18 +165,24 @@ function DeathKnight:BloodGlowCooldowns()
 	local talents = fd.talents;
 	local covenantId = fd.covenant.covenantId;
 
-	local abominationLimbReady = DeathKnight.db.abominationLimbAsCooldown and covenantId == Necrolord and cooldown[BL.AbominationLimb].ready;
-	local dancingRuneWeaponReady = DeathKnight.db.bloodDancingRuneWeaponAsCooldown and cooldown[BL.DancingRuneWeapon].ready;
+	local abominationLimbReady = covenantId == Necrolord and cooldown[COMMON.AbominationLimb].ready;
+	local abominationLimbTalentReady = talents[COMMON.AbominationLimbTalent] and cooldown[COMMON.AbominationLimbTalent].ready;
+	local dancingRuneWeaponReady = talents[BL.DancingRuneWeapon] and cooldown[BL.DancingRuneWeapon].ready;
+	local boneStormReady = talents[BL.Bonestorm] and cooldown[BL.Bonestorm].ready;
+	local empowerRuneweaponReady = talents[COMMON.EmpowerRuneWeapon] and cooldown[COMMON.EmpowerRuneWeapon].ready;
 
 	if DeathKnight.db.alwaysGlowCooldowns then
-		MaxDps:GlowCooldown(BL.AbominationLimb, abominationLimbReady);
+		MaxDps:GlowCooldown(COMMON.AbominationLimb, abominationLimbReady);
+		MaxDps:GlowCooldown(COMMON.AbominationLimbTalent, abominationLimbTalentReady);
+		MaxDps:GlowCooldown(BL.Bonestorm, boneStormReady);
```

Comment:
This is a very nieche cooldown, I wouldn't add it to be honest

Reply:
Bonestorm? I don't see any reason not to glow it if they have it.

Reply:
It isn't recommended in any rotations or guides, it dumps your whole Runic Power and is, if ever, only useful in AOE.